src/res/translation/wininstaller/installerlng.nsi

```diff
@@ -19,3 +19,6 @@
 
 !insertmacro MUI_LANGUAGE "French"
 !include "${ROOT_PATH}\src\res\translation\wininstaller\fr.nsi"
+
+!insertmacro MUI_LANGUAGE "Spanish"
+!include "${ROOT_PATH}\src\res\translation\wininstaller\sp.nsi"
```

Comment:
Just a small question here: why did you choose sp.nsi not es.nsi?

Reply:
Good question... Changed.

Reply:
Great!